src/data/content/design-system/componenti/buttons.yaml

```diff
@@ -82,24 +82,25 @@ tabs:
             title: Come usarlo
             headingLevel: 2
             text: |
-              - Usa la variante primary per valorizzare l’azione principale
-              - Usa la variante secondary per valorizzare l’azione secondaria
-              - Usa pulsanti disabilitati con moderazione, assicurandoti che per l’utente sia chiaro come attivarli
+              - Usa la variante `primary` per valorizzare l’azione principale.
+              - Usa la variante `secondary` per valorizzare l’azione secondaria.
               - Per comunicare senza ambiguità la gerarchia delle azioni generate dai pulsanti, usa le varianti di grandezza e tipologia.
               - Usa un pulsante con icona per aggiungere un’informazione visiva all’interazione (es. pulsante di accesso ad area riservata)
 
               ### Attenzione a
-              - Non usare più di un pulsante primary nello stesso contesto di azione
-              - Non usare pulsanti primary per azioni di valore secondario
-              - Non usare i pulsanti all’interno di paragrafi di testo
+              - Usare pulsanti disabilitati con moderazione, assicurandoti che per l’utente sia chiaro come attivarli.
```

Comment:
```suggestion
              - Usare pulsanti disabilitati con moderazione, assicurandoti che per l’utente sia chiaro come attivarli.
```

Reply:
questa frase formulata così a me pare quasi il contrario di ciò che dovrebbe essere a una prima lettura... cc @danielenole @Fupete 

forse sarebbe + comprensibile "Attenzione a...non esagerare con la quantità di pulsanti disabilitati; assicurati in ogni caso che per l'utente sia chiaro come attivarli"